zcash_client_sqlite/src/chain.rs

```diff
@@ -26,10 +26,14 @@ pub mod migrations;
 /// Starting at `from_height`, the `with_row` callback is invoked with each block retrieved from
 /// the backing store. If the `limit` value provided is `None`, all blocks are traversed up to the
 /// maximum height.
+///
+/// # Panics
+///
+/// Panics if the provided `limit` value exceeds the range of a u32
```

Comment:
Rather than panicking, we could just saturate; it doesn't really matter either way as `u32::MAX` is the maximum block height.